src/drive/web/modules/filelist/FileList.jsx

```diff
@@ -7,11 +7,20 @@ import FileListBody from 'drive/web/modules/filelist/FileListBody'
 
 import styles from 'drive/styles/filelist'
 
-const FileList = ({ canSort, fileActions, ...rest }) => (
+const FileList = ({
+  canSort,
+  fileActions,
+  withSelectionCheckbox = true,
```

Comment:
During a previous refactor, we stopped respecting the `withSelectionCheckbox ` prop — but not all view should have it.